### DISCUSSION TITLE
emergency_stepの左右方向転倒時にどちらの脚を最初に上げるのか

### DISCUSSION THREAD

**Discussion by Naoki-Hiraoka** [YutaKojio/hrpsys-base#19] [Q&A] [answered]
emergency_stepの左右方向転倒時にどちらの脚を最初に上げるのかは、現状TODOですか?
https://github.com/YutaKojio/hrpsys-base/blob/c40cffe9818d33cae39ed961b99f5562c907c828/rtc/AutoBalancer/AutoBalancer.cpp#L692-L695
前後方向については実装されていそうです。

**Comment by YutaKojio** [comment, marked as answer]
あ，そうだった．やってたつもりだったけど，横方向に押された時はどっちの足上げるのが正解かわからず消してました．
現状は，足を前後に開いた状態で前後に外乱が加わった時は，外乱方向じゃない足を先に上げるようになっているだけです．

つまり，左右方向転倒時はTODOです．
